syne_tune/callbacks/tensorboard_callback.py

```diff
@@ -163,7 +163,7 @@ def __getstate__(self):
         return state
 
     def __setstate__(self, state):
-        super().__init__(
+        self.__init__(
```

Comment:
I am not familiar with how to code `__setstate__`. But what is the difference here? A comment would be nice, because since most of us do not understand this "Python magic", we just copy and paste.

I am very suspicious about calling the constructor for an already existing class (what else is `self`), so what I did in the scheduler and searcher hierarchy is to define method that make more sense to me.

I note there is also `syne_tune.blackbox_repository.BlackboxRepositoryBackend` which has the same code, could you change that as well?

Reply:
OK, the code in `BlackboxRepositoryBackend` is correct, it is supposed to call the super-class constructor.